Docs: Add separate fundamentals topic on notification policies (#69174)
grafana/grafana: docs/sources/alerting/manage-notifications/create-notification-policy.md

# Manage notification policies

Notification policies determine how alerts are routed to contact points.

Policies have a tree structure, where each policy can have one or more nested policies. Each policy, except for the default policy, can also match specific alert labels.

Each alert is evaluated by the default policy and subsequently by each nested policy.

If the **Continue matching subsequent sibling nodes** option is enabled for a nested policy, then evaluation continues even after one or more matches. A parent policy’s configuration settings and contact point information govern the behavior of an alert that does not match any of the nested policies. A default policy governs any alert that does not match a nested policy.

> You can configure Grafana managed notification policies as well as notification policies for an external Alertmanager data source.

Reviewer: ```suggestion
You can configure Grafana-managed notification policies as well as notification policies for an external Alertmanager data source.
```

> See also our guide on the [fundamentals of Notification Policies]({{< relref "../fundamentals/notification-policies/index.md" >}}).

Reviewer: ```suggestion
For more information on notification policies, see [fundamentals of Notification Policies]({{< relref "../fundamentals/notification-policies/index.md" >}}).
```
